# Move base node parameters. For full documentation of the parameters in this file, please see
#
#  http://www.ros.org/wiki/move_base
#
shutdown_costmaps: false

controller_frequency: 2.5
controller_patience: 0.0

planner_frequency: 0.0
planner_patience: 5.0

oscillation_timeout: 0.0
oscillation_distance: 0.2

# local planner - default is trajectory rollout
base_local_planner: "neuro_local_planner_wrapper/NeuroLocalPlannerWrapper"

base_global_planner: "global_planner/GlobalPlanner" #alternatives: global_planner/GlobalPlanner, navfn/NavfnROS, carrot_planner/CarrotPlanner


#We plan to integrate recovery behaviors for turtlebot but currently those belong to gopher and still have to be adapted.
## recovery behaviors; we avoid spinning, but we need a fall-back replanning
#recovery_behavior_enabled: true

recovery_behaviors:
  - name: 'neuro_fake_recovery'
    type: 'neuro_fake_recovery/NeuroFakeRecovery'

